### PR TITLE
Add links arrows

### PR DESCRIPTION
**Pull request by k-jano**
This PR introduce links arrows in rca dashboard

arrow_links

### CODE REVIEW

File: src/components/RCA.js [hunk 1]
```diff
@@ -236,7 +247,8 @@ export class RCA extends React.Component {
     const link = linkGroup.append("line")
       .data(links, d => [d.source, d.target])
       .join('line')
-      .attr('class', d => {return d.fault ? 'link fault' : 'link';});
+      .attr('class', d => {return d.fault ? 'link fault' : 'link';})
```
Comment (bzurkowski) [inline src/components/RCA.js:250]: What is `d.fault` in this line?

**Review by bzurkowski — COMMENTED** [review]
@k-jano One question.

File: src/components/RCA.js (hunk 1 above)
Comment (k-jano) [reply src/components/RCA.js:250]: This is ``link`` object. Variable ``d`` is a common convention used in d3.js. Example of ``d`` object
```
{
  "id": "1",
  "source": {
    "id": "4",
    "kind": "alert",
    "origin": "prometheus",
    "properties": {
      "name": "ServiceFailure1"
    },
    "index": 3,
    "x": 30.4140380706331,
    "y": 27.22177798935603,
    "vy": 0.00001773177850070695,
    "vx": -0.000009516850564199232
  },
  "target": {
    "id": "3",
    "kind": "alert",
    "origin": "prometheus",
    "properties": {
      "name": "PodFailure1"
    },
    "index": 2,
    "x": -45.9888297678861,
    "y": -46.98554243748834,
    "vy": 0.00003110514940390333,
    "vx": -0.00001734190878249983
  },
  "properties": {
    "strength": 0.95,
    "foo": "bar"
  },
  "fault": true,
  "index": 0
}
```

Comment (bzurkowski) [reply src/components/RCA.js:250]: I do not recall the `fault` property in the fault trajectory API payload. Is it D3-specific or maybe introduced in a post-processing stage?

Comment (k-jano) [reply src/components/RCA.js:250]: This property is added in post-processing stage. We are detecting fault nodes - these ones, which are between two alerts and add property to link element. Through that we are able to mark it in visualization by red color.

https://github.com/openrca/orca-ui/blob/master/src/components/GraphUtils.js#L97

**Review by bzurkowski — APPROVED** [review]
@k-jano LGTM , let's merge 😉

**Pull request merged**